virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -9,6 +9,13 @@ Before you install {VirtProductName}, ensure that your {product-title} cluster m
 * Your cluster must be installed on
 xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal] infrastructure with Red Hat Enterprise Linux CoreOS workers.
 
+* Additionally, your cluster must use xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[installer-provisioned infrastructure] and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[deploy machine health checks] to maintain high availability (HA).
+
+[NOTE]
+====
+In {VirtProductName} clusters installed using installer-provisioned infrastructure, if a node fails the MachineHealthCheck and becomes unavailable to the cluster, a `RunStrategy` determines what happens next, depending on a series of conditions. See xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-about-runstrategies-vms_virt-create-vms[About RunStrategies for virtual machines] for more detailed information about RunStrategies.
```

Comment:
Please consider the following text, I think that it is clearer.

In {VirtProductName} clusters installed using installer-provisioned infrastructure **and with MachineHealthCheck properly configured**, if a node fails the MachineHealthCheck and becomes unavailable to the cluster **it would be recycled. What happens next with the VMs that had run on that node depend** on a series of conditions. See xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-about-runstrategies-vms_virt-create-vms[About RunStrategies for virtual machines] for more detailed information about RunStrategies.

Reply:
Changed suggested wording just a little:

"In OpenShift Virtualization clusters installed using installer-provisioned infrastructure and with MachineHealthCheck properly configured, if a node fails the MachineHealthCheck and becomes unavailable to the cluster, it is recycled. **What happens next with VMs that ran on the failed node depends on a series of conditions.** See About RunStrategies for virtual machines for more detailed information about the potential outcomes and how RunStrategies affect those outcomes."

Reply:
Thanks.